config/default-spinnaker-local.yml

```diff
@@ -15,6 +15,19 @@ providers:
       # Store actual credentials in $HOME/.aws/credentials. See spinnaker.yml for
       # more information, including alternatives.
 
+  azure:
+    # If you want to deploy some services to Microsoft Azure (azure),
+    # set enabled and provide primary credentials for deploying.
+    # Enabling azure is independent of other providers.
+    enabled: false
```

Comment:
Since you added the environment variables to default spinnaker, these should be ${SPINNAKER_AZURE_ENABLED:false} and similar reference for the region. Otherwise you arent utilizing those environment variables.